### ISSUE TITLE
Create wrapper around OpenFin cli

### ISSUE THREAD

**Issue by BenLambertNcl** [symphonyoss/ContainerJS#212]
Currently, we are passing the `app.json` file directly to the OpenFin cli. In future, we might want to pass different config to Electron/OpenFin whilst keeping the same app.json file. To do this, we will need to wrap the OpenFin cli with our own cli. An example where this is useful is if we want the initial OpenFin window to display hidden (to act like the Electron main process) but keep the Electron initial window visible.

To do this, we would need to output a json file from our cli, and then pass the file path to the OpenFin cli as it doesn't support passing config parameters directly.

**Comment by ColinEberhardt** [comment]
Another advantage is that currently we have to respect the OpenFin requirements for app.json, which require certain fields. Here is a minimal example:

```
{
    "startup_app": {
        "name": "Hello World",
        "url": "http://localhost:8080/index.html",
        "uuid": "hello-world",
        "autoShow": true
    },
    "runtime": {
        "version": "stable"
    }
}
```

We don't have any need for the `runtime` property, or `uuid`

**Comment by BenLambertNcl** [comment]
Closed by #228
